driverkit/utils/cleanup

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+user=poiana
+
+# Assuming the drivers versions of Falco eBPF probes and Falco kernel modules are always in pair.
+# For this reason, obtain all the version by only querying the Falco kernel modules versions.
+get_bintray_versions() {
+    IFS=$'\n' read -r -d '' -a all < <(curl -s -u "${user}:${pass}" --header "Content-Type: application/json" https://api.bintray.com/packages/falcosecurity/driver/kernel-module | jq -r '.versions | .[]')
+}
+
+filterout_versions() {
+    for i in "${!all[@]}";
+    do
+        printf "%s\t%s\n" "$i" "${all[$i]}"
+        if [[ ! " ${keep[*]}" =~  ${all[$i]} ]]; then
+            printf " \tremoving..."
+            JFROG_CLI_LOG_LEVEL=DEBUG jfrog bt vd --user "${user}" --key "${pass}" "falcosecurity/driver/kernel-module/${all[$i]}" --quiet
```

Comment:
Here we could download (or directly move the driver) to S3 for archivation as per proposal in https://github.com/falcosecurity/falco/pull/1375.


Waiting for feedback.

Should we block this task until we have S3?
Should we implement the archivation step in a second phase?